pipelines/broad/dna_seq/germline/single_sample/wgs/WholeGenomeGermlineSingleSample.options.json

```diff
@@ -2,6 +2,6 @@
   "read_from_cache": true,
   "write_to_cache": true,
   "default_runtime_attributes": {
-    "zones": "us-central1-a us-central1-b us-east1-d us-central1-c us-central1-f us-east1-c"
+    "zones": "us-central1-a us-central1-b us-east1-d us-central1-f"
```

Comment:
I think we did this because we were running into resource problems in a couple of zones (couldn't find an N2 in that host?).  Since I don't remember, I think I'll revert the change.